connexion/exceptions.py

```diff
@@ -82,13 +82,13 @@ class OAuthProblem(Unauthorized):
     pass
 
 
-class OAuthResponseProblem(Unauthorized):
+class OAuthResponseProblem(OAuthProblem):
     def __init__(self, token_response, **kwargs):
         self.token_response = token_response
         super(OAuthResponseProblem, self).__init__(**kwargs)
 
 
-class OAuthScopeProblem(Forbidden):
+class OAuthScopeProblem(Forbidden, OAuthProblem):
```

Comment:
It seems a bit odd to mixin `OAuthProblem` (and by proxy `Unauthorized`) to `OAuthScopeProblem`.
I wonder if it doesn't make more sense to except both `OAuthProblem` and `OAuthScopeProblem` in 
[aiohttp_api.py](https://github.com/zalando/connexion/blob/a0ac0fd13f6857bba2563fbdda55404cfa69dde3/connexion/apis/aiohttp_api.py#L33) instead.
What do you think?